cekit/builders/docker_builder.py

```diff
@@ -32,14 +31,8 @@
 except ImportError:
     pass
 
-try:
-    # Docker Python library, the old one
-    import docker  # noqa: F811
-    from docker.client import Client as APIClientClass  # noqa: F811
-except ImportError:
-    pass
-
 ANSI_ESCAPE = re.compile(r"\x1B\[[0-?]*[ -/]*[@-~]")
+DOCKER_API_VERSION = "1.35"
```

Comment:
Is this the current API version? Do we have any knowledge about supported API versions in the current library?

What I'm thinking is: we should not make it recent if the library supports older versions as well. The reason is that not all packages/systems around the world are up to date. Another thing is that we probably don't use any features from new APIs.

Reply:
We last changed it in 2002 responding to https://github.com/cekit/cekit/pull/774 when the original version (like it **currently** is in Behave (cc @jmtd)) was too old. The compatibility matrix is https://docs.docker.com/engine/api/#api-version-matrix ; unfortunately they don't give dates but the next API change (1.36) came with 18.02 and that was 6 years ago. So IMHO we're still picking a rather old version. The API version I nearly picked (1.41) came from 20.10 and is 3 years ago.

Reply:
@jmtd @goldmann  Do we know what versions of Docker Engine are available for RHEL, Debian, etc? 
GHActions use 1.43 API I think from looking at a recent run

Reply:
> @jmtd @goldmann Do we know what versions of Docker Engine are available for RHEL, Debian, etc? GHActions use 1.43 API I think from looking at a recent run

Technically, there is no Docker in RHEL/CentOS Stream. So I imagine the Docker's `docker-engine` packages provide latest versions.